src/components/Home/JoinList.jsx

```diff
@@ -18,7 +18,7 @@ const JoinList = ({
         <label htmlFor="share-token">Share Token:</label>
         <input
           id="share-token"
-          type="text"
+          type="search"
```

Comment:
I did not know this causes the built-in clear input button! That's awesome!

Reply:
I know this is previous functionality too but the <datalist> element is also something I haven't seen before and I think it's a great way to accomplish this functionality instead of building a separate selector and you can combine it with the input box all in one place